src/utils/validateProject.js

```diff
@@ -41,5 +42,8 @@ export default async function validateProject(project: Project) {
    *     <More Project checks here>
    */
 
-  return projectIsValid;
+  return {
+    invalidMessages,
+    projectIsValid
```

Comment:
Don't you need to update `install` now as well?

Reply:
yes,, I am planning to call validate command from install to keep single flow for project validation.

Reply:
I wouldn't call the command, I would call the util. Calling the command means the validate command is limited by whatever the install command wants to output